src/main/resources/assets/integrateddynamics/lang/en_US.lang

```diff
@@ -807,6 +809,95 @@ operator.operators.integrateddynamics.cast.name=Cast
 operator.operators.integrateddynamics.cast.basename=Cast
 operator.operators.integrateddynamics.cast.tooltip=Cast %s to %s
 
+## Config categories
+config.integrateddynamics.item=Item settings
+config.integrateddynamics.item.tooltip=Change settings of items
```

Comment:
Same comment as for CyclopsCore, these tooltips have no real added value, so they can be removed IMO.